index.js

```diff
@@ -0,0 +1,98 @@
+let myLibrary = [];
+const auth = document.getElementById('author');
+const title = document.getElementById('title');
+const pages = document.getElementById('pages');
+const read = document.getElementById('read');
+const btn = document.getElementById('submit');
+const table = document.getElementById('table');
+
+function getBooks() {
```

Comment:
## Implement function with ES6 arrow function
- [ ] Implement this `getBooks` function with ES6 arrow function
```suggestion
const getBooks = () => {
```

After the book has been created, there is no way to display the books in the `localStorage` as there is no way javascript can iterate over the array properly. This is the error we currently have. 👇 
![library-book-creation-error](https://user-images.githubusercontent.com/22345961/106790514-bab32f00-6664-11eb-9651-ac281e3e0685.png)
- [ ] Kindly fix this issue, thank you!